connection.go

```diff
@@ -735,6 +735,10 @@ func buildSnowflakeConn(ctx context.Context, config Config) (*snowflakeConn, err
 		queryContextCache:   (&queryContextCache{}).init(),
 		currentTimeProvider: defaultTimeProvider,
 	}
+	err := initEasyLogging(config.ClientConfigFile)
```

Comment:
Do we officially call it easy logging? Or is it just client config(uration)?

Reply:
We do logging reconfiguration so I named it accordingly. It is also named in such a way in other drivers.
Recently I also added more descriptive error information using "easy logging" phrase, because it relates to a particular feature.